techradar/radars/admin.py

```diff
@@ -1,7 +1,17 @@
 from django.contrib import admin
-from .models import Radar
+from .models import Radar, Segment, Technology
```

Comment:
Jestem fanem absolutnych importów w projektach django: 

```suggestion
from techradar.radars.models import Radar, Segment, Technology
```